code_samples/api/public_php_api/src/Command/UpdateContentCommand.php

```diff
@@ -54,7 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $contentDraft = $this->contentService->updateContent($contentDraft->versionInfo, $contentUpdateStruct);
         $this->contentService->publishVersion($contentDraft->versionInfo);
 
-        $output->writeln('Content item ' . $contentId . ' updated with new name: ' . $newName);
+        $output->writeln('content item ' . $contentId . ' updated with new name: ' . $newName);
```

Comment:
I wouldn't change this one as this the first world of a sentence

```suggestion
        $output->writeln('Content item ' . $contentId . ' updated with new name: ' . $newName);
```